layers/+emacs/org/packages.el

```diff
@@ -588,6 +590,19 @@ Headline^^            Visit entry^^               Filter^^                    Da
       (kbd "M-SPC") 'spacemacs/org-agenda-transient-state/body
       (kbd "s-M-SPC") 'spacemacs/org-agenda-transient-state/body)))
 
+(defun org/init-org-notify ()
+  (use-package org-notify
+    :config
+    (org-notify-add 'appt
+                    '(:time "-1s" :period "20s" :duration 10
+                            :actions (-message -ding))
+                    '(:time "15m" :period "2m" :duration 100
+                            :actions -notify)
+                    '(:time "2h" :period "5m" :actions -message)
+                    '(:time "3d" :actions -email))
+    (when org-start-notification-daemon-on-startup
+      (org-notify-start))))
```

Comment:
What happens when you call org-notifiy-start twice? I am just wondering if we have to start that once for every buffer or once for an emacs session?

Reply:
This function should only be called once for an Emacs session.